homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -89,6 +89,7 @@ async def _async_show_user_form(
             errors=errors or {},
         )
 
+    # flake8: noqa: C901
```

Comment:
Function is too complex (C901)
Sorry for that, but I did not want to refactor the existing code.
I think it is better to address the change in a specific pull request

Reply:
I don't think this comment is needed any more, let's remove it.